build.sbt

```diff
@@ -91,6 +92,9 @@ lazy val seleniumJSEnv: Project = project.
         "com.novocode" % "junit-interface" % "0.11" % "test"
     ),
 
+    sources in Test +=
+      baseDirectory.value.getParentFile / "project/TestCapabilities.scala",
```

Comment:
We usually do this the other way around. We put the source file in the normal source folder of the project (so in `scalajs-env-selenium/src/test/scala/...`) and we add the `sources +=` in `project/plugins.sbt`. It has at least one concrete advantage: IDEs are happier with it.